src/hooks/screen/useChatScreenHook.ts

```diff
@@ -7,6 +7,7 @@ import {useRecoilState} from 'recoil';
 import {useRoute} from '@react-navigation/core';
 import {v4 as uuid} from 'uuid';
 
+import {Platform} from 'react-native';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `Platform` from 'react-native' is a good addition to differentiate between Android and other platforms. This will help in handling platform-specific logic.
```diff
+ import {Platform} from 'react-native';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->